test/e2e/lib/components/site-editor-component.js

```diff
@@ -254,9 +254,16 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 			await driverHelper.clickWhenClickable( this.driver, blockAppenderLocator );
 		} );
 
+		// @todo Remove this fallback and its reference below in the `By.css` call
```

Comment:
@david-szabo97 Would it be possible to share this function between the two classes? Maybe by extracting it to an external helper module? I noticed they are pretty much the same, but there's a small difference -- the fact that this one wraps the `blockAppenderLocator`'s `By.css` call in the `runInCanvas`. Why is this needed? Thanks!

```

Reply:
The editor content is in an iframe in the Site Editor. In the Post Editor, it isn't. We wrap it in `runInCanvas` to make sure we run that part of the code in the editor's iframe content.

We can declare an empty function `runInCanvas` function in the PostEditor's Component class. Then we can pass the editor to the helper function. After that, we can remove the function from both components.

Reply:
Makes sense! I'll do that as a follow-up PR at some point this or next week as time allows 👍🏻